pom.xml

```diff
@@ -66,9 +67,20 @@
 
```

Comment:
Since you are releasing internally to MSH, you might remove the prior sections: `<repositories>`, `<pluginRepositories>`, `<distributionManagement>`. They might be benign, but I'm wondering if they might override values in the parent pom and cause issues when you run a release.

Reply:
+1 to this

Reply:
This `<scm>` section is actually okay to remove. It used to be needed but the new build process doesn't need it to complete a maven release. However, it's benign to leave in there if you want to.